bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableDataGrpcClient.java

```diff
@@ -157,6 +177,17 @@ public Empty mutateRow(MutateRowRequest request) throws ServiceException {
   }
 
   @Override
+  public MutateRowsResponse mutateRows(MutateRowsRequest request) throws ServiceException {
+    return clientCallService.blockingUnaryCall(createMutateRowsCall(request), request);
+  }
+
+  @Override
+  public ListenableFuture<MutateRowsResponse> mutateRowsAsync(MutateRowsRequest request) {
+    expandPoolIfNecessary(this.bigtableOptions.getChannelCount());
```

Comment:
The pool expansion is a  performance enhancement and not a major functionality, so it's questionable.  We can discuss the severity offline.